library/src/main/java/com/chuckerteam/chucker/api/BodyDecoder.kt

```diff
@@ -0,0 +1,29 @@
+package com.chuckerteam.chucker.api
+
+import okhttp3.Request
+import okhttp3.Response
+import okio.ByteString
+import okio.IOException
+
+/**
+ * Decodes HTTP request and response bodies to human–readable texts.
+ */
+public interface BodyDecoder {
+    /**
+     * Returns a text representation of [body] that will be displayed in Chucker UI transaction,
+     * or `null` if [request] cannot be handled by this decoder. [Body][body] is no longer than
```

Comment:
Looks like a typo with 2 `body`

Reply:
I believe is actually valid KDoc where you can specify the display text for a link 🤔

Reply:
Yes, it is deliberate. It allows to link to `body` property and start the sentence with a capital letter.